.travis.yml

```diff
@@ -8,8 +8,8 @@ notifications:
     secure: j3YP9TjiIcMRy2mvunF1AHBOFnz2H7mZAFVbHPBNkAjMCwSdBNvLpn33qv6ybr02c5snBDJTs0P70RJ/mh3YsqwnIeloQD9HUfnndKQD6ujxx1QWRI/lVDW4pfVRQEuPsXdW/3AiqxrSG5BS4thiyc3vj3LpnodHwNMUT+Nlmq0=
 services:
   - mysql
-  - postgresql
 addons:
+  postgresql: "9.4"
```

Comment:
It would be good if there is a standard about the versions we are supported among every database and test against different versions.